messages/en.json

```diff
@@ -796,6 +797,69 @@
     "villagergame_win": "Game over! The villagers come to their senses and realize there are actually no wolves, and live in harmony forevermore. Everybody wins.",
     "villagergame_nope": "Game over! The villagers decided incorrectly that there are actually no wolves, allowing the wolves to slaughter the remainder of them in their sleep with impunity.",
     "stop_bot_ingame_safeguard": "Warning: A game is currently running. If you want to {what} the bot anyway, use \"{prefix}{cmd} -force\".",
+    "fwarn_usage": "Usage: fwarn list|view|add|del|set|help. See fwarn help <subcommand> for more details.",
+    "warn_usage": "Usage: warn list|view|ack|help. See warn help <subcommand> for more details.",
+    "fwarn_list_syntax": "Usage: fwarn list [-all] [nick[!user@host]|=account] [page]",
+    "fwarn_view_syntax": "Usage: fwarn view <id>",
+    "fwarn_del_syntax": "Usage: fwarn del <id>",
+    "fwarn_set_syntax": "Usage: fwarn set <id> [~expiry] [reason] [| notes]",
+    "fwarn_help_syntax": "Usage: fwarn help <subcommand>",
+    "warn_list_syntax": "Usage: warn list [-all] [page]",
+    "warn_view_syntax": "Usage: warn view <id>",
+    "warn_ack_syntax": "Usage: warn ack <id>",
+    "warn_help_syntax": "Uwage: warn help <subcommand>",
+    "fwarn_add_syntax": "Usage: fwarn add <nick[!user@host]|=account> [@]<points> [~expiry] [sanctions] <:reason> [| notes]",
+    "fwarn_page_invalid": "Invalid page, must be a number 1 or greater.",
+    "fwarn_points_invalid": "Invalid points, must be a number above 0.",
+    "fwarn_expiry_invalid": "Invalid expiration amount, must be a number above 0 or 'never' for a warning that never expires.",
+    "fwarn_expiry_invalid_suffix": "Invalid expiration suffix, must use either d, h, or m.",
+    "fwarn_cannot_add": "Cannot add warning, double-check your parameters (the nick might be wrong or you are not joined to the channel).",
+    "fwarn_added": "Added warning {0}.",
+    "fwarn_done": "Done.",
+    "fwarn_sanction_invalid": "Invalid sanction, can be either deny or stasis.",
+    "fwarn_stasis_invalid": "Invalid stasis amount, specify sanction as stasis=number.",
+    "fwarn_deny_invalid": "Invalid denied commands, specify sanction as deny=command,command,command without spaces.",
```

Comment:
It's meant to be read as `command, command, command` without spaces.

On 16 June 2016 22:59:23 CEST, Emanuel Barry <notifications@github.com> wrote:
>> +    "fwarn_help_syntax": "Usage: fwarn help <subcommand>",
>> +    "warn_list_syntax": "Usage: warn list [-all] [page]",
>> +    "warn_view_syntax": "Usage: warn view <id>",
>> +    "warn_ack_syntax": "Usage: warn ack <id>",
>> +    "warn_help_syntax": "Uwage: warn help <subcommand>",
>> +    "fwarn_add_syntax": "Usage: fwarn add
><nick[!user@host]|=account> [@]<points> [~expiry] [sanctions] <:reason>
>[| notes]",
>> +    "fwarn_page_invalid": "Invalid page, must be a number 1 or
>greater.",
>> +    "fwarn_points_invalid": "Invalid points, must be a number above
>0.",
>> +    "fwarn_expiry_invalid": "Invalid expiration amount, must be a
>number above 0 or 'never' for a warning that never expires.",
>> +    "fwarn_expiry_invalid_suffix": "Invalid expiration suffix, must
>use either d, h, or m.",
>> +    "fwarn_cannot_add": "Cannot add warning, double-check your
>parameters (the nick might be wrong or you are not joined to the
>channel).",
>> +    "fwarn_added": "Added warning {0}.",
>> +    "fwarn_done": "Done.",
>> +    "fwarn_sanction_invalid": "Invalid sanction, can be either deny
>or stasis.",
>> +    "fwarn_stasis_invalid": "Invalid stasis amount, specify sanction
>as stasis=number.",
>> +    "fwarn_deny_invalid": "Invalid denied commands, specify sanction
>as deny=command,command,command without spaces.",
>
>"command without spaces" ... ?
>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/lykoss/lykos/pull/225/files/16a0a28e360087b90b0640c724fe2b0039f81697#r67423006

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.